### PR TITLE
VAN.get_person() - Fix MyVoters Error

### PR DESCRIPTION
**Pull request by jburchard**
The default fields returned from `VAN.get_person()` are only returned with EveryAction, not MyVoters. This PR removes them when the database is set to MyVoters.

Addresses issue #310.

### CODE REVIEW

File: parsons/ngpvan/people.py [hunk 1]
```diff
@@ -360,6 +360,12 @@ def get_person(self, id, id_type='vanid', expand_fields=[
 
         expand_fields = ','.join([json_format.arg_format(f) for f in expand_fields])
 
+        # Removing the fields that are not returned in MyVoters
+        NOT_IN_MYVOTERS = ['codes', 'contribution_history', 'organization_roles']
+
+        if self.db == 0:
```
Comment (eliotst) [inline parsons/ngpvan/people.py:366]: I think you want this to be `self.connection.db_code` ... `self.db` the value passed in to the `__init__` by the user, which appears to be either `MyVoters`, `MyMembers`, `MyCampaign`, or `EveryAction`. `self.connection.db_code` is the numeric value.

**Comment by jburchard** [comment]
Ready for re-review.

File: parsons/ngpvan/people.py [hunk 2]
```diff
@@ -360,6 +360,12 @@ def get_person(self, id, id_type='vanid', expand_fields=[
 
         expand_fields = ','.join([json_format.arg_format(f) for f in expand_fields])
 
+        # Removing the fields that are not returned in MyVoters
+        NOT_IN_MYVOTERS = ['codes', 'contribution_history', 'organization_roles']
+
+        if self.connection.db == 0:
```
Comment (ydamit) [inline parsons/ngpvan/people.py:366]: I think you still want this to be `self.connection.db_code == 0` OR `self.connection.db == 'MyVoters'` (I prefer the former).

**Comment by jburchard** [comment]
Ready for final review. Sorry for all of the back and forth on this.

**Pull request merged**